pandas/tests/io/pytables/test_subclass.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+
+from pandas import DataFrame, Series
+import pandas._testing as tm
+from pandas.tests.io.pytables.common import ensure_clean_path
+
+from pandas.io.pytables import HDFStore, read_hdf
+
+
+class TestHDFStoreSubclass:
```

Comment:
in `pandas._testing` we have `SubclassedSeries` and `SubclassedDataFrame`. Should we be using those here?

Reply:
Thank you for pointing this out!
Changed that.